### DISCUSSION TITLE
Should we move the "other" into "builtin" folder?

### DISCUSSION THREAD

**Discussion by Neo23x0** [SigmaHQ/sigma#2584] [General]
Hey guys, 

shouldn't we move the folder `rules/windows/other` to `rules/windows/builtin/other` as it contains only log sources that are builtin in Windows, like the Print Service or Windows Defender. 

@frack113 @thomaspatzke 

![Screenshot 2022-01-20 at 08 49 26](https://user-images.githubusercontent.com/2851492/150295747-8826a8fb-04eb-43a8-b271-23f7b519f2c3.png)

**Comment by frack113** [comment]
Yes 
Why not move `msexchange` to application as you need to install it ?
Other are builtin channel to enable.

**Reply by Neo23x0** [reply]
MS Exchange should appear under /Microsoft/Windows 
I would handle it as `builtin`. I consider `application` as 3rd party applications that are not part of the OS. 
But this is not a strong opinion. If everybody else thinks differently, I can align with that.

![Screenshot 2022-01-20 at 09 52 30](https://user-images.githubusercontent.com/2851492/150305214-2bc93f84-0594-453a-a8e1-0a73e5b127aa.png)

**Comment by frack113** [comment]
Let's start with a simple rule: 
- events below "windows" in windows folder 
- the others in "application" folder

**Comment by thomaspatzke** [comment]
Yes!
